source/Mlos.Model.Services/OptimizationProblem.cs

```diff
@@ -40,5 +40,20 @@ public OptimizationProblem()
         {
             Objectives = new List<OptimizationObjective>();
         }
+
+        public OptimizationProblem(Hypergrid parameterSpace, Hypergrid objectiveSpace, OptimizationObjective[] objectives)
+        {
+            ParameterSpace = parameterSpace;
+            ObjectiveSpace = objectiveSpace;
+            Objectives = new List<OptimizationObjective>(objectives);
+        }
+
+        public OptimizationProblem(Hypergrid parameterSpace, Hypergrid contextSpace, Hypergrid objectiveSpace, OptimizationObjective[] objectives)

```

Comment:
IEnumerable<OptimizationObjective>

Reply:
[zaporter/grpc 4a74b7b5] Refactored to use C# syntax tricks